src/Symfony/Component/Form/FormEvents.php

```diff
@@ -12,7 +12,7 @@
 
 /**
  * To learn more about how form events work check the documentation
- * entry at {@link http://symfony.com/doc/any/components/form/form_events.html}
+ * entry at {@link http://symfony.com/doc/any/components/form/form_events.html}.
```

Comment:
A short description is only allowed to be one line long by some parsers. That means our descriptions looks like this:

short:
```
To learn more about how form events work check the documentation
```

long:
```
entry at {@link http://symfony.com/doc/any/components/form/form_events.html}.

To learn how to dynamically modify forms using events check the cookbook
entry at {@link http://symfony.com/doc/any/cookbook/form/dynamic_form_modification.html}.
```

Do you see what I mean now. With or without the dot, this docblock needs correcting first before continuing.